cmd/chart.go

```diff
@@ -76,6 +78,8 @@ func newChartMoveCmd() *cobra.Command {
 	f.UintVar(&retries, "retries", defaultRetries, "number of times to retry push operations")
 	f.StringVar(&output, "out", "*.relocated.tgz", "name of the resulting chart")
 
+	f.StringVar(&toArchive, "to-archive", "", "save the chart and all its dependencies to and offline archive tarball")
```

Comment:
BTW, since we are building a binary in each commit (which we should not), I'd comment out this line so the flag for a WIP feature is not exposed

Reply:
That is a problem, if I do that tests will fail.

Maybe I should comment the tests as well.

Reply:
I will comment both as the last commit fix on this PR

Reply:
Done